flac.md

```diff
@@ -222,9 +222,9 @@ Data     | Description
 `u(36)`  | Total samples in stream. 'Samples' means inter-channel sample, i.e. one second of 44.1 kHz audio will have 44100 samples regardless of the number of channels. A value of zero here means the number of total samples is unknown.
 `u(128)` | MD5 signature of the unencoded audio data. This allows the decoder to determine if an error exists in the audio data even when the error does not result in an invalid bitstream. A value of `0` signifies that the value is not known.
 
-The minimum block size is excluding the last block of a FLAC file, which may be smaller. If the minimum block size is equal to the maximum block size, the file contains a fixed block size stream. Note that in case of a stream with a variable block size, the actual maximum block size MAY be smaller than the maximum block size listed in the streaminfo block, and the actual smallest block size excluding the last block MAY be larger than the minimum block size listed in the streaminfo block. This is because the encoder has to write these fields before receiving any input audio data, and cannot know beforehand what block sizes it will use, only between what bounds these will be chosen.
+FLAC specifies a minimum block size of 16 and a maximum block size of 65535. This does not apply to the block size of the last frame in a stream, because the block size of that frame must be able to accomodate for the length of the audio data the file contains. As the last block is excluded from the block sizes stated in the streaminfo block, the bit patterns corresponding to the numbers 0-15 in the minimum block size and maximum block size fields are invalid.
```

Comment:
```suggestion
The minimum block size and the maximum block size MUST be in the 16-65535 range. The minimum block size MUST be equal to or less than the maximum block size.

Any frame but the last one MUST have a block size equal to or greater than the minimum block size and MUST have a block size equal to or lesser than the maximum block size. The last frame MUST have a block size equal to or lesser than the maximum block size, it does not have to comply to the minimum block size because the block size of that frame must be able to accommodate for the length of the audio data the stream contains. 
```

Just a suggestion, not sure it is really better, yours is also fine.

Reply:
I think it is an improvement, more formal.